magicctapipe/scripts/lst1_magic/lst1_magic_create_irf.py

```diff
@@ -18,14 +18,14 @@
 diffuse data is available for test MCs.
 
 There are four different event types with which the IRFs are created.
-The "hardware" type is supposed for the hardware trigger between LST-1
+The "hardware" type stands for the hardware trigger between LST
 and MAGIC, allowing for the events of all the telescope combinations.
-The "software(_only_3tel)" types are supposed for the software event
-coincidence with LST-mono and MAGIC-stereo observations, allowing for
-only the events triggering both M1 and M2. The "software" type allows
-for the events of the any 2-tel combinations except the MAGIC-stereo
-combination at the moment. The "software_only_3tel" type allows for only
-the events of the 3-tel combination. The "magic_only" type allows for
+The "software_3tels_or_more" type stands for the software event
```

Comment:
if we modify the types following my earlier comment they need to be updated here as well.

Reply:
changed